packages/gatsby-image/src/index.js

```diff
@@ -208,7 +208,8 @@ class Image extends React.Component {
 
   handleRef(ref) {
     if (this.state.IOSupported && ref) {
-      this.cleanUpListeners = listenToIntersections(ref, () => {
+      const rootMargin = this.props.rootMargin || `200px`
```

Comment:
Maybe use defaultProps here?

Reply:
Also would it make sense to make this prop be a number _or_ a string, and if it's a number we presume `px`?

Reply:
missed the default props, on it.